src/app/containers/StoryPromo/index.stories.jsx

```diff
@@ -68,4 +70,5 @@ storiesOf('Containers|Story Promo/AMP', module)
   .add('audio promo - amp', () => getAmpStoryPromo(audioFixture))
   .add('video promo - amp', () => getAmpStoryPromo(videoFixture))
```

Comment:
Perhaps change these two or the ones below this as well. I don't see why there's a distinction for amp only in the strings for the first two and not the rest. I think either all the strings should have `- amp` or none of them should have it as it can be understood in the code. I haven't run this so it might be clearer adding `- amp` to all of them.

Reply:
Done now :) Thank you!

Reply:
![yay](https://media2.giphy.com/media/Ge86XF8AVY1KE/giphy.gif)